### PR TITLE
Remove outdated Python set up instructions

### PR DESCRIPTION
**Pull request by mayurva**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @mayurva, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by mayurva** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @mayurva has just signed the CLA. Thanks, and we look forward to your contribution.

File: test/README.md [hunk 1]
```diff
@@ -52,29 +53,19 @@ pyenv local cr  # auto-activate virtualenv when entering this directory
...
-wget https://bootstrap.pypa.io/get-pip.py
-python get-pip.py
-pip install virtualenv
-pip install virtualenvwrapper
+    If you are using a Mac for dev setup, you can remove `cx_freeze` as a dependency from [dev-requirements.txt](../dev-requirements.txt). This package is only needed for packaging ClusterRunner binary, and can be ignored for other development purposes.
 
-# Add the next two lines to your shell startup file (.bashrc, .profile, etc.)
-export WORKON_HOME=$HOME/.virtualenvs  # add to .bashrc
-source virtualenvwrapper.sh  # add to .bashrc
+1. Functional tests time out on OSX:
```
Comment (ethomas2) [inline test/README.md]: change this 1. to a 2. :)

I realize markdown will render this as 2, but it bugs me

File: test/README.md [hunk 2]
```diff
@@ -35,15 +36,15 @@ This is the preferred method since installing Python 3.4 via Homebrew is no long
 brew update
 brew install pyenv
 
-# Add pyenv init to your shell startup file 
-echo 'eval "$(pyenv init -)"' >> ~/.bash_profile  # replace .bash_profile with whatever you use (.bashrc, .profile, etc.) 
+# Add pyenv init to your shell startup file
+echo 'eval "$(pyenv init -)"' >> ~/.bash_profile  # replace .bash_profile with whatever you use (.bashrc, .profile, etc.)
 
 # Install Python 3.4
 pyenv install 3.4.5  # use latest 3.4.X
```
Comment (cmcginty) [inline test/README.md]: Might as well update this to 3.4.8 until we can move to newer python version.

Comment (mayurva) [reply test/README.md]: Updated..

File: test/README.md (hunk 1 above)
Comment (mayurva) [reply test/README.md]: Spoke to @ethomas2 offline. Didn't strongly suggest me to change it. Keeping it as is.

**Comment by mayurva** [comment]
@ethomas2, @cmcginty and I found that the timeout errors are happening due to the way DNS is configured on my machine. Given that and we possibly removing dependency on cx_freeze, I don't think we need this pull request anymore.

**Pull request merged**